l10n_it_withholding_tax/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 
 {
     'name': 'Italian Withholding Tax',
-    'version': '12.0.2.0.1',
+    'version': '12.0.2.0.2',
```

Comment:
Non è necessario modificare la versione, in quanto verrà poi fatto automaticamente da @OCA-git-bot